mariadb/scale-mariadb-server/scale-mariadb-server.sh

```diff
@@ -28,13 +28,20 @@ az monitor metrics list \
 --metric storage_used \
```

Comment:
__Required__: `az monitor metrics list` fails:

```
ERROR: az monitor metrics list: error: the following arguments are required: --resource
```

Scripts are __required to be tested prior to submission.__ https://review.docs.microsoft.com/en-us/help/contribute/contribute-scripts-pr-process?branch=master#test-script-changes